taskfiles/docker.yml

```diff
@@ -1,8 +1,17 @@
 version: '3'
 
 tasks:
+
+  download-task:
+    cmds:
+    - sh -c "$(curl --location https://taskfile.dev/install.sh)" --
```

Comment:
why not always put it in /usr/local/bin/task or /usr/bin - if docker, seems ok, so that way it is in the $PATH

Reply:
this is done with the goal of having `bin/task` be available when bind-mounting the workspace folder in order to share it with a container. The `docker:task` task runs Task within a docker container, so having it in the host's `PATH` only works for the outer task, but not for the `task` invocation within docker container... (those were too many "task"s)